client/src/views/Simulation/components/ParametersPanel.vue

```diff
@@ -403,6 +404,10 @@
   .parameter.hidden {
     opacity: 0.5;
   }
+
+  .parameter.highlighted {
+    border: 1px var(--selection) solid;
```

Comment:
It will be better to add a border to parameter to avoid the jump when being highlighted:

https://user-images.githubusercontent.com/636801/127937403-34d311a5-ca2d-4a18-9cea-69132de1354b.mov

doing something like this:
```CSS
.parameter {
  ...
  border: 1px solid transparent;
}

.parameter.highlighted {
  border-color: var(--selection);
}
```

Reply:
Great point, changed.